pyrovision/models/rexnet.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the GNU Affero General Public License version 3.
+# See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
+
+from holocron.models.rexnet import ReXNet
+from typing import Dict, Any
+from pyrovision.models.utils import cnn_model
+from holocron.models.utils import load_pretrained_params
+
+__all__ = ['rexnet1_0x', 'rexnet1_3x', 'rexnet1_5x', 'rexnet2_0x', 'rexnet2_2x']
+
+
+default_cfgs: Dict[str, Dict[str, Any]] = {
+    'rexnet1_0x': {'width_mult': 1.0, 'depth_mult': 1.0,
+                   'url': 'https://github.com/frgfm/Holocron/releases/download/v0.1.2/rexnet1_0x_224-ab7b9733.pth'},
+    'rexnet1_3x': {'width_mult': 1.3, 'depth_mult': 1.0,
+                   'url': 'https://github.com/frgfm/Holocron/releases/download/v0.1.2/rexnet1_3x_224-95479104.pth'},
+    'rexnet1_5x': {'width_mult': 1.5, 'depth_mult': 1.0,
+                   'url': 'https://github.com/frgfm/Holocron/releases/download/v0.1.2/rexnet1_5x_224-c42a16ac.pth'},
+    'rexnet2_0x': {'width_mult': 2.0, 'depth_mult': 1.0,
+                   'url': 'https://github.com/frgfm/Holocron/releases/download/v0.1.2/rexnet2_0x_224-c8802402.pth'},
+    'rexnet2_2x': {'width_mult': 2.2, 'depth_mult': 1.0,
+                   'url': None},
+}
```

Comment:
The architecture is unchanged vs. "rexnet1_0x" & the others in Holocron. It would be best to use the functions from there, the only diff is that we map pretrained differently:
- pretrained = True on our side should point to the checkpoint you added (thus passing pretrained=False to the Holocron rexnet functions)
- if pretrained=False, perhaps we can pass an additional args "imagenette_pretrained" which will pass its value to the "pretrained" of Holocron rexnet functions

Reply:
yes makes sens :)